src/stcal/ramp_fitting/likely_fit.py

```diff
@@ -62,22 +47,48 @@ def likely_ramp_fit(ramp_data, readnoise_2d, gain_2d):
 
     readtimes = get_readtimes(ramp_data)
 
+    # Check for a zeroframe
+    if ramp_data.zeroframe is not None:
+        # Read time for the zeroframe is the first entry in the first group
+        rt_zero = readtimes[0][0]
+
+        # Remove the zero readtime from the first group
+        readtimes[0] = readtimes[0][1:]
+
+        # Add the zero readtime as a separate entry in the beginning of the array
+        readtimes.insert(0, np.array([rt_zero]))
+
     covar = Covar(readtimes)
     integ_class = IntegInfo(nints, nrows, ncols)
 
     readnoise_2d = readnoise_2d / SQRT2
 
     for integ in range(nints):
-        data = ramp_data.data[integ, :, :, :]
-        gdq = ramp_data.groupdq[integ, :, :, :].copy()
-        pdq = ramp_data.pixeldq[:, :].copy()
+        data = ramp_data.data[integ]
+        gdq = ramp_data.groupdq[integ].copy()
+        pdq = ramp_data.pixeldq.copy()
+
+        # Use the zeroframe if available
+        if ramp_data.zeroframe is not None:
```

Comment:
```suggestion
        if hasattr(ramp_data, 'zeroframe') and ramp_data.zeroframe is not None:
```
As before, to make it easier to use this with Roman